README.md

```diff
@@ -1,12 +1,12 @@
-## KIN Distributor
-This project lets you run a small localhost server that communicates with stellar network.
-It will create a KIN asset on Stellar Test Network, related to the issuer seed in `config.json` file.
-You will have a rest-api `sendkin` where you need to pass address as url parameter<br/>
-and the issuer will send 1K KIN to the specific address.
+# KIN Distributor
+This project lets you run a small local server that communicates with Stellar TestNet.
+It will create a KIN asset, related to the issuer seed in `config.json` file.
+You will have a function that submit REST HTTP request `sendkin` where you need to pass address as url parameter<br/>
+and the issuer will send 1K KIN to the specific address.<br/>
```

Comment:
remove `</br>`, i don't understand why it's there at all. it breaks the line unexpectedly and looks weird

Reply:
`1000 KIN` instead of `1K KIN`

Reply:
Instead of the "you will have a function" line, write this:

> Use the `sendkin()` function which accepts a URL address as a parameter. This will issue 1000 KIN to the specified address.

Also functions in go use camel case so the function name should probably be `sendKin()`